index.html

```diff
@@ -2,18 +2,154 @@
 <hmtl>
   <head>
     <title>YouTube</title>
+    <link rel="stylesheet" href="styles/index.css" />
   </head>
   <body>
     <!-- Navbar here-->
     <header>
-      <nav class="navbar"></nav>
+      <nav class="navbar">
+        <div class="grid-item grid-item1">
+          <img class="logo" src="img/Youtubelogo.png" alt="logo" />
+        </div>
+
+        <div class="grid-item grid-item2">
+          <input type="text" name="search" placeholder="Search" />
+          <button type="button"><img src="img/search.png" /></button>
+        </div>
+
+        <div class="grid-item grid-item3">
+          <a href="https://www.youtube.com/" class="nav_links"
+            ><img src="img/notification.png" />
+          </a>
+          <a href="#" class="nav_links"><img src="img/camera.png" /></a>
+          <a href="#" class="nav_links"><img src="img/menu.png" /></a>
+        </div>
+      </nav>
     </header>
 
     <main class="main-content">
       <!-- Main video here -->
       <section class="video_big"></section>
       <!-- Side bar here -->
```

Comment:
## Please add the large video section
- Following these requirements 

- [ ] It is displayed ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_displayed_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
- [ ] It shows the video title ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_title_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
- [ ] It shows the number of views ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_views_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
- [ ] It has like and dislike icons ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_like_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
- [ ] It has a share button ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_share_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
- [ ] It shows the avatar of the user who uploaded the video ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_avatar_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
- [ ] It has a subscribe button ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_subscribe_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))

- This section
![image](https://user-images.githubusercontent.com/24902525/90462607-c323a780-e0ce-11ea-8bbc-7bbf48029b7c.png)